e2e/utils/zetacore.go

```diff
@@ -448,3 +481,56 @@ func WaitForZetaBlocks(
 		}
 	}
 }
+
+// WaitForZRC20BalanceChange waits for the zrc20 balance of the given address to change by the given delta amount
```

Comment:
What happen if the balance never change? Does the test get stuck?
Should have a low retry, as the belance is support to change very fast